### ISSUE TITLE
Does not install into current directory on Windows

### ISSUE THREAD

**Issue by jmartsch** [kittn/generator-kittn#121]
Kittn does not recognize the correct path where the files should be created.
I did this in my console:
```
mkdir kittn
cd kittn
yo kittn
```
Then the installer says  `Please give the project a name (without Spaces) (xampphtdocs)`
it seems to ignore the windows directory separator, because the folder or project name should be either xampp/htdocs/kittn or the kittn directory should be created automatically in xampp/htdocs/

Could you provide a fix for this? I am willing to test.

**Comment by CinKon** [comment]
The suggested projectname `(xampphtdocs)` is just a suggestion for the projectname ;)
It doesn't have anything to do with the file path, this isn't needed for the yo-installer. Just type some projectname you like (`my-great-awesome-project` or something)

**Comment by jmartsch** [comment]
The problem still exists if I type in a project-name, I know that xampphtdocs is just a suggestion.
I typed in a project name, but after that the files are not installed into the directory I am in, nor does it install the files in a directory with the projectname I provided.
It installs all files in my root directory xampp/htdocs/
There are now several folders gulpfile, src, etc. in my root, which should go into `my-great-awesome-project` instead.
